docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -823,6 +804,22 @@ these pieces of data from it. It then performs the following actions:
 
 </Admonition>
 
+<Details
```

Comment:
I'm not sure Step 4 is the best place for this. Should we add these instructions where the `Details` box that this PR removes used to be?

Reply:
That sounds good to me, have moved it over.